vulnerabilities/render/index.js

```diff
@@ -0,0 +1,15 @@
+'use strict';
+const express = require('express');
+const api = express.Router();
+
+api.get('/', (req, res) => {
+  res.render(`${__dirname}/views/index`);
```

Comment:
a nitpick but can we just render the same template in both cases, and just not pass the options?